specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3535,6 +3535,125 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/ipAddressAggregates": {
+      "get": {
+        "tags": [
+          "Reports"
+        ],
+        "description": "Gets the ipAddressAggregates for a given service.",
+        "x-ms-examples": {
+          "services_listIpAddressAggregates": {
+            "$ref": "./examples/IpAddressAggregates.json"
+          }
+        },
+        "operationId": "services_listIpAddressAggregates",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "skiptoken",
```

Comment:
is it an actual token or just numeric row number? Is it related to the continuationtoken?
if just number, you can use standard skip. If it is the same as continuationtoken returned, please keep the name same to maintain consistency.

Reply:
It's an actual token, IE:  
ipAddressAggregates?api-version=2014-01-01&**skiptoken=1!76!MWQ5MDY4ZTQtMG**
I think the name part has to match right?  In the code, we're using "skiptoken", so the name here has to be the same.